### ISSUE TITLE
Arrow keys don't work when I SSH out

### ISSUE THREAD

**Issue by peterloron** [cmderdev/cmder#114]
I have cmder installed and working well on my Win7 machine. However, when I SSH to a CentOS 6.5 machine (putty link 0.62), the arrow keys do not work on the remote machine. 

When I use the cygwin shell and ssh from there, it works well. Not sure if this is a cmdr issue or a putty issue. In both cases, the remote machine is showing 'xterm' as the terminal type.

**Comment by GuilhermeBorges** [comment]
Same issue here, drives me crazy ;/

**Comment by ksuszka** [comment]
I've got similiar issue using docker inside cmder. If I log into docker container interactive session arrow keys don't work. AFAIK docker uses SSH under the hood so it is probably the same issue.

**Comment by shaquille-young** [comment]
I've got similiar issue with recent version

**Comment by shaquille-young** [comment]
@peterloron @GuilhermeBorges @ksuszka It seems that add "set term=cons25" to .vimrc will resolve this problem.

Reference: [Fix arrow keys that display A B C D on remote shell](http://vim.wikia.com/wiki/Fix_arrow_keys_that_display_A_B_C_D_on_remote_shell)

**Comment by erikjasiak** [comment]
Any updates or are you accepting PR's on this issue (and maybe a lead on where to look?)

I'm trying to get back into using windows as a dev platform for a few things, with a *lot* of backend ssh connections into linux vm's and remote servers.   In my testing of terminals, this is hands down the best looking most enjoyable - even more than the new Win 10 native one.  But the inability to arrow-up/down through command history once I'm ssh remote is a dealbreaker.

Still would rather try to help fix it instead of complain - or does anyone have a workaround?  Tried changing remote TERM type to every value of terminfo I know about, but I'm guessing it's more of an escape sequence hook buried a little deeper.

(And finally yes, I've already updated my ComEMU backend - that seems to be a separate issue.)
